### PR TITLE
Turn off randomization for leak detection bots.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: dev/bots/test.dart [hunk 1]
```diff
@@ -187,6 +187,12 @@ String get shuffleSeed {
   return _shuffleSeed!;
 }
 
+final bool _isRandomizationOff = () {
+  // The values can be different, one is compile time, another is run time.
```
Comment (keyonghan) [inline dev/bots/test.dart:191]: Remove?

**Review by keyonghan — APPROVED** [review]
LGTM with nit.

File: dev/bots/test.dart [hunk 2]
```diff
@@ -187,6 +187,9 @@ String get shuffleSeed {
   return _shuffleSeed!;
 }
 
+final bool _isRandomizationOff = const bool.fromEnvironment('TEST_RANDOMIZATION_OFF') ||
```
Comment (christopherfujino) [inline dev/bots/test.dart:190]: bool.fromEnvironment doesn't check for environment variables, it checks for dart defines. You want to check `Platform.environment['TEST_RANDOMIZATION_OFF']` or something like that (please double check the docs)

**Review by christopherfujino — CHANGES_REQUESTED** [review]

File: dev/bots/test.dart (hunk 2 above)
Comment (polina-c) [reply dev/bots/test.dart:190]: Platform.environment is checked by next line.

Comment (polina-c) [reply dev/bots/test.dart:190]: But we can add this later if needed.

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**